package.json

```diff
@@ -34,6 +34,9 @@
     "lint-staged": "8.0.4",
     "prettier": "1.14.3"
   },
+  "engines": {
+    "npm": ">5.2.0"
```

Comment:
since you're in there, mind adding the other necessary engines? (https://jira.we.co/browse/GROW-6372)